.github/dependabot.yml

```diff
@@ -32,3 +32,10 @@ updates:
     directory: "/typespec/"
     schedule:
       interval: "weekly"
+    groups:
+      angular:
```

Comment:
This should say `typespec` or something other than `angular` :) Other than that, looks good.

Reply:
oops! updated.